sherpa/astro/flux.py

```diff
@@ -563,23 +571,18 @@ def print_sample_result(title, arg):
         iflx = numpy.zeros(size)  # intrinsic/unabsorbed flux
         thawedpars = [par for par in fit.model.pars if not par.frozen]
 
-        logger = logging.getLogger("sherpa")
-        orig_log_level = logger.level
-
         mystat = []
         for nn in range(size):
-            logger.setLevel(logging.ERROR)
             session.set_source(id, orig_source)
-            logger.setLevel(orig_log_level)
             oflx[nn] = mysim[nn, 0]
             for ii in range(len(thawedpars)):
                 val = mysim[nn, ii + 1]
                 session.set_par(thawedpars[ii].fullname, val)
             session.set_source(id, modelcomponent)
-            iflx[nn] = session.calc_energy_flux(lo, hi)
+            iflx[nn] = session.calc_energy_flux(lo=lo, hi=hi, id=id)
```

Comment:
I know you are just fixing this routine, but I do note that it has no docstring. Since you just made fixes, you probably know better than anyone what it does and what the parameters are. Could you add a short doctring?
*I would have commented in line 531, but github does not let me add comments in lines that are not changed in this PR.*

Reply:
I have a whole lot of commits in #754 and I'm pretty sure I add documentation in them.

I just checked and it turns out I haven't added documentation. In part because the work isn't done yet to work out what I want this routine to do... Later PRs make more-significant changes to this routine so I think it makes sense to add doc changes then.